src/main/resources/application.yml

```diff
@@ -106,5 +106,5 @@ management:
         project-id: ceres-dev-222017
 
 security:
-  whitelistedIpRange: "192.168.0.1/32"
-  whitelistedRoles: "COMPUTE_DEFAULT"
+  whitelistedIpRange: "10.40.0.0/14"
```

Comment:
May not want to give any default here since this code is a public repo.

Reply:
It should be private address space but I will look into it.

Reply:
Yeah, 10.* is private anyway, so not too important either way.